bayes_opt/bayesian_optimization.py

```diff
@@ -233,7 +277,13 @@ def suggest(self, utility_function):
         return self._space.array_to_params(suggestion)
 
     def _prime_queue(self, init_points):
-        """Make sure there's something in the queue at the very beginning."""
+        """Ensure the queue is not empty.
+
+        Parameters
+        ----------
+        init_points: int
+            Number of parameters to prime the queue with.
+        """
```

Comment:
Is this *really* what this function does? The term "prime" gives me the impression that this is preparing/driving the queue to the start point.

Reply:
The code is pretty short and fits here:
```python


    def _prime_queue(self, init_points):
        """Ensure the queue is not empty.

        Parameters
        ----------
        init_points: int
            Number of parameters to prime the queue with.
        """
        if self._queue.empty and self._space.empty:
            init_points = max(init_points, 1)

        for _ in range(init_points):
            self._queue.add(self._space.random_sample())
```

So what happens is a) if the queue is empty and there are no points registered to the target space it samples at least one point randomly from the space and adds it to the queue. I'm open to changing the description (or function name) to something that is more intuitive. Any suggestions?

> The term "prime" gives me the impression that this is preparing/driving the queue to the start point.

Not sure what exactly you mean by this, I think the `prime` is used here as a stand-in for `prepare`.

Reply:
I think it's fine